src/store/foodprint-context.tsx

```diff
@@ -136,6 +147,14 @@ const FoodprintContextProvider: React.FC = (props) => {
     setIngredients(array);
   }
 
+  const searchResultResetHandler = () => {
```

Comment:
why do we need a handler for this?

Reply:
Handler was made preemptively, but is not fully necessary.